sdk/trace/sampling.go

```diff
@@ -30,11 +35,46 @@ type SamplingParameters struct {
 	SpanID          core.SpanID
 	Name            string
 	HasRemoteParent bool
+	Kind            api.SpanKind
+	Attributes      []core.KeyValue
+	Links           []api.Link
+}
+
+// SamplingDecision indicates whether a span is recorded and sampled.
+type SamplingDecision uint8
+
+// Valid sampling decisions
+const (
+	NotRecord SamplingDecision = iota
+	Record
+	RecordAndSampled
```

Comment:
I'd say it should be "RecordedAndSampled" or "RecordAndSample", but I see that the spec specified the decision name like that…

Reply:
Yeah, I'm not a fan either, but I think sticking to the spec is the thing to do here.  At least until someone changes the spec :)